kumascript/macros/LearnSidebar.ejs

```diff
@@ -1881,6 +1893,24 @@ var text = mdn.localStringMap({
             <li><a href="<%=baseURL%>Accessibility/Accessibility_troubleshooting"><%=text['Assessment_Accessibility_troubleshooting']%></a></li>
         </ol>
     </details>
+  </li>
+   <li><a href="<%=baseURL%>Web_Performance"><strong><%=text['Web_Performance_—_Making_websites_fast_and_responsive']%></a></strong></li>
```

Comment:
'Web_Performance_—_Making_websites_fast_and_responsive' doesn't match 'Web_performance_making_websites_fast_and_responsive' from line 171, so we don't see an entry here.

Also, the indentation of this line is off.